### PR TITLE
Update to files from multilog

### PR DESCRIPTION
**Pull request by arvedes**
Update files to current multilog output as discussed with @aalbino2 for implementation of #3

### CODE REVIEW

File: melt_czochralski_Dadzis/base_classes.schema.archive.yaml [hunk 1]
```diff
@@ -40,81 +29,14 @@ definitions:
                   eln:
                     component: ReferenceEditQuantity  
     Experiment: 
-      base_sections: 
-      - nomad.datamodel.metainfo.eln.ElnBaseSection
-      - '#/Activity'
+      base_section:  '#/Entity'
```
Comment (aalbino2) [inline melt_czochralski_Dadzis/base_classes.schema.archive.yaml:32]: Why Experiment is now Entity? It should be either Activity or Workflow

File: melt_czochralski_Dadzis/base_classes.schema.archive.yaml [hunk 2]
```diff
@@ -137,58 +59,17 @@ definitions:
                 # hide: ['children', 'parents']
                 # hide: ['children', 'parents', institute]
                 hide: []
-        geometry:
-          section: '#/Geometry'
+        composition:
+          repeats: true
+          #m_annotations:
+          #  eln:
+          section: '#/EntityAndAmount'
```
Comment (aalbino2) [inline melt_czochralski_Dadzis/base_classes.schema.archive.yaml:66]: These Material classes are not yet established, are you using some of them in your schema?

File: melt_czochralski_Dadzis/multilog_eln.archive.yaml [hunk 3]
```diff
@@ -1,27 +1,42 @@
 definitions:
```
Comment (aalbino2) [inline melt_czochralski_Dadzis/multilog_eln.archive.yaml:1]: Is the melt_czochralski.schema.archive.yaml file still used?

**Review by aalbino2 — CHANGES_REQUESTED** [review]

File: melt_czochralski_Dadzis/multilog_eln.archive.yaml (hunk 3 above)
Comment (arvedes) [reply melt_czochralski_Dadzis/multilog_eln.archive.yaml:1]: It's now all included in multilog_eln.archive.yaml

File: melt_czochralski_Dadzis/base_classes.schema.archive.yaml (hunk 2 above)
Comment (arvedes) [reply melt_czochralski_Dadzis/base_classes.schema.archive.yaml:66]: I don't think so, I think you told me that I had to include this file in my upload. But maybe that's outdated?

File: melt_czochralski_Dadzis/base_classes.schema.archive.yaml (hunk 1 above)
Comment (arvedes) [reply melt_czochralski_Dadzis/base_classes.schema.archive.yaml:32]: This is the file currently used in multilog, I just copied it from here. If the present version in this repo is better I'll replace it!

File: melt_czochralski_Dadzis/multilog_eln.archive.yaml (hunk 3 above)
Comment (aalbino2) [reply melt_czochralski_Dadzis/multilog_eln.archive.yaml:1]: Okay, let's keep it like this for now. If we will experience issue then we can always come back to separate schema and data files again. 
Whenever you will upload a file containing a schema, you will have it available in nomad for further experiments you want to record within the same data structure. The idea was to put all the schemas in one folder and separate them from the instances you create out of it. But I believe you will figure out your better routine along the way

File: melt_czochralski_Dadzis/base_classes.schema.archive.yaml (hunk 1 above)
Comment (aalbino2) [reply melt_czochralski_Dadzis/base_classes.schema.archive.yaml:32]: Yes, I didn't mention that the most updated is in the base_classes folder, so could you replace this with the new one please? You should also test if the upload is still successful with the new base_classes.schema.archive.yaml

File: melt_czochralski_Dadzis/base_classes.schema.archive.yaml (hunk 2 above)
Comment (aalbino2) [reply melt_czochralski_Dadzis/base_classes.schema.archive.yaml:66]: Same as previous comment

**Review by aalbino2 — APPROVED** [review]

**Pull request merged**